modules/upi-provisioning-new-hosts-in-a-upi-cluster.adoc

```diff
@@ -68,26 +179,34 @@ $ oc create -f bmh.yaml
 .Example output
 [source,terminal]
 ----
-secret/worker1-bmc created
-baremetalhost.metal3.io/worker1 created
+secret/openshift-worker-<num>-network-config-secret created
+secret/openshift-worker-<num>-bmc-secret created
+baremetalhost.metal3.io/openshift-worker-<num> created
 ----
++
+where:
 
-. Approve all certificate signing requests (CSRs).
+<num>:: Specifies the worker node number.
 
-.. Verify that the provisioning state of the host is `provisioned` by running the following command:
+. Inspect the bare-metal node by running the following command:
 +
 [source,terminal]
 ----
-$ oc get bmh -A
+$ oc -n openshift-machine-api get bmh openshift-worker-<num>
 ----
 +
+where:
+
+<num>:: Specifies the worker node number.
++
 .Example output
```

Comment:
This is embedding under the description list instead of the step. If you want to use a desc list here, put it in an open block to fix the indent. (or you could single-line it as mentioned above, or maybe omit this "where" entirely since num is defined right before)

Reply:
Updated as suggested.